code/game/objects/items/miscellaneous.dm

```diff
@@ -376,3 +376,67 @@
 	name = "living lube delivery beacon"
 	default_name = "Offensive"
 	mob_choice = /mob/living/simple_animal/hostile/retaliate/clown/lube
+
+//	-----------------
+//	Detective loadout
+//	-----------------
+
+/obj/item/choice_beacon/detective
+	name = "detective beacon"
+	desc = "There are many ways of investigating crime, pick yours."
+
+/obj/item/choice_beacon/detective/generate_display_names()
+	var/static/list/detective_item_list
+	if(!detective_item_list)
+		detective_item_list = list()
+		var/list/templist = typesof(/obj/item/storage/box/detectivebeacon) //we have to convert type = name to name = type, how lovely!
+		for(var/V in templist)
+			var/atom/A = V
+			detective_item_list[initial(A.name)] = A
+	return detective_item_list
+
+/obj/item/storage/box/detectivebeacon
+	name = "The Private Investigator"
+	desc = "Gritty, rude, and answers all questions in a sarcastic tone. The Private Investigator likes to rely more on his own senses and intuition, instead of this fancy new technology. Contains your standard gear, and an olfaction injector."
+
+/obj/item/storage/box/detectivebeacon/PopulateContents()
+	new /obj/item/clothing/suit/det_suit/noir(src)
+	new /obj/item/clothing/under/rank/security/detective/grey(src)
+	new /obj/item/clothing/head/fedora (src)
+	new /obj/item/ammo_box/c38(src)
+	new /obj/item/dnainjector/privatedick(src)
+
+/obj/item/storage/box/detectivebeacon/undercover
+	name = "The Undercover Agent"
+	desc = "No, I don't know of any syndicate activity, mister Rick Detectiveson! The Undercover Agent Kit provides you with additional gear required to disguise yourself as a regular assistant."
+
+/obj/item/storage/box/detectivebeacon/undercover/PopulateContents()
+	new /obj/item/implanter/undercover(src)
+	new /obj/item/radio/headset/headset_undercover(src)
+	new /obj/item/clothing/under/color/grey(src)
+	new /obj/item/card/assistant_disguise_card(src)
+	new /obj/item/ammo_box/c38/trac(src)
+
+/obj/item/storage/box/detectivebeacon/coroner
+	name = "The Coroner"
+	desc = "The dead tell may tales, if you know how to liste. For a detective with a little medical expertise, the Coroner Kit provides you with plenty of medical tools to support and investigate trails that have not gone cold yet."
+
+/obj/item/storage/box/detectivebeacon/coroner/PopulateContents()
+	new /obj/item/clothing/glasses/hud/medsec(src)
+	new /obj/item/storage/firstaid/advanced(src)
+	new /obj/item/clothing/suit/toggle/labcoat(src)
+	new /obj/item/sensor_device(src)
+	new /obj/item/locator(src)
+	new /obj/item/ammo_box/c38/match/bouncy(src)
+
+/obj/item/storage/box/detectivebeacon/fastresponder
+	name = "The Fast Responder"
+	desc = "For being the first in the line of combat, the fast responder kit equipts you with deputy gear, a detective vest and plenty of robust gear to handle arrests."
```

Comment:
Detective is not even real security why would he need to be fast responder

Reply:
> plenty of robust gear to handle arrests."

Ick

Reply:
Remove this option entirely. Combat det is bad.

Reply:
Noted.

Reply:
They're not freely available. You have to ask other player to get it.

Reply:
> A lot of stuff (armor, sechud, jackboots) are already freely available to the Detective.
Outside of cult murderboning or nuke ops, the detective would be banned for running around with this gear